ci/recipe/run_test.sh

```diff
@@ -3,3 +3,7 @@
 git clone https://github.com/biocore/deblur deblur-upstream
 nosetests --with-doctest deblur-upstream
 rm -rf deblur-upstream
+
+git clone https://github.com/qiime2/q2-deblur q2_deblur
```

Comment:
No need to git-clone, we want to fail if we're missing package_data (which is what the above workaround was for)
```suggestion

```

Reply:
okay gotcha, that makes sense - thanks @ebolyen!